Frontend/UpFluxAutomationTest/LoginTest/EngineerLoginTest.cs

```diff
@@ -0,0 +1,43 @@
+using NUnit.Framework;
+using UpFluxAutomation.Flows;
+using UpFluxAutomationTest.TestBase;
+using UpFluxAutomation.Helpers;
+using UpFluxAutomation.Steps;
+using System;
+using UpFluxAutomation.Abstractions;
+
+namespace UpFluxAutomation.Tests
+{
+    [TestFixture]
+    public class EngineerLoginTests : TestBase
+    {
+        [Test]
+        public async Task TestEngineerLogin()
+        {
+            try
+            {
+                // Create and initialize EngineerData
```

Comment:
Will this be updated like we discussed to use only token to login?

Reply:
Yes, the test can be updated to use only the token for login. but that's need to be modified in the UI, As I'm testing what is available i need to execute the test like this way, either that, the project is setup to handle everything Dynamically(avoiding dependencies), that is why  takes me a while just to built the logic.

Reply:
Okay, good job